styles/styles.css

```
@@ -46,17 +419,41 @@
 .arrow {
     background-image: url("/img/hotels/arrow.svg");
     background-repeat: no-repeat;
-    background-color: brown;
 }
 .mode-tile {
     background-image: url("/img/hotels/mode-tile.svg");
     background-repeat: no-repeat;
+    border: 2px solid #E6E6E6;
+    background-position: center;
+    box-sizing: border-box;
+    border-radius: 4px;
+    display: inline-block;
+    width: 48px;
+    height: 48px;
 }
 .mode-slideshow {
     background-image: url("/img/hotels/mode-slideshow.svg");
     background-repeat: no-repeat;
+    border: 2px solid #E6E6E6;
+    background-position: center;
+    box-sizing: border-box;
+    border-radius: 4px;
+    display: inline-block;
+    width: 48px;
+    height: 48px;
 }
 .mode-list {
     background-image: url("/img/hotels/mode-list.svg");
     background-repeat: no-repeat;
+    border: 2px solid #E6E6E6;
+    background-position: center;
+    box-sizing: border-box;
+    border-radius: 4px;
+    display: inline-block;
+    width: 48px;
+    height: 48px;
+}
+.filter {
+    background-image: url("/img/hotels/catalog-background.jpg");
```

Все адреса должны быть относительными
```suggestion
    background-image: url("../img/hotels/catalog-background.jpg");
```